ReactiveXComponentTest/UnitTests/RabbitMqUnitTests/RabbitMqSubscriberTest.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using NFluent;
+using NSubstitute;
+using NUnit.Framework;
+using RabbitMQ.Client;
+using ReactiveXComponent.RabbitMq;
+using ReactiveXComponent.RabbitMQ;
+
+namespace ReactiveXComponentTest.UnitTests.RabbitMqUnitTests
+{
+    [TestFixture]
+    public class RabbitMqSubscriberTest: RabbitMqTestBase
```

Comment:
Using Throws or DoesNotThrow is a useful Assert. But making tests only based on this Assert a a bit weird.